src/Dapr.Actors/Runtime/ActorHost.cs

```diff
@@ -60,8 +122,11 @@ public sealed class ActorHost
         /// </summary>
         public IActorProxyFactory ProxyFactory { get; }
 
-        internal DaprStateProvider StateProvider { get; set; }
+        /// <summary>
+        /// Gets the <see cref="ActorTimerManager" />.
+        /// </summary>
+        public ActorTimerManager TimerManager { get; }
 
-        internal IDaprInteractor DaprInteractor { get; set; }
+        internal DaprStateProvider StateProvider { get; set; }
```

Comment:
The state provider needs to get the same treatment. It's *somewhat* possible to mock the state provider today.